CONTRIBUTING.md

```diff
@@ -66,6 +66,90 @@ Clone the repository to your local machine from the Azure DevOps endpoint.
+
+- In the 'RetrospectiveExtension.Frontend' folder, create the 'vss-extension-dev.json' file using the template.
+
+- Update the 'webpack.config.js' to enable source maps. Set the devtool property to `inline-source-map`. Also set devServer.https to true and devServer.port to 3000.
+
+  ```js
+  module.exports = {
+    devtool: 'inline-source-map',
+    devServer: {
+      https: true,
+      port: 3000,
+      static: {
+        directory: path.join(__dirname),
+      }
+    },
+  ...
+  ```
+
+- Set `output.publicPath` to `/dist/` in the webpack.config.json file. This will allow webpack to serve files from `localhost:3000/dist`.
+
+  ```js
+  module.exports = {
+      output: {
+        publicPath: "/dist/"
+        // ...
+      }
+      // ..
+  };
+  ```
+
+- In order for webpack to copy HTML files from the `src` folder to the `dist` folder, add the following lines to the webpack.config.json file.
+
+  ```js
+  module.exports = {
+    plugins: [
+      new CopyWebpackPlugin({
+      patterns: [{from: "**/*.html", context: "src"}]
+    }),
+    // ...
+  };
+  ```
+  
+- Set up a debug configuration for VS Code that launches Firefox with the correct path mappings. Add a path mapping with `url` set to `webpack:///` and path set to `${workspaceFolder}/RetrospectiveExtension.Frontend/`. Also set the reAttach property on the configuration to true to avoid restarting Fiefox every time you debug
+  
+  ```json
+  {
+    "version": "0.2.0",
+    "configurations": [
+      {
+        "name": "Launch Firefox",
+        "type": "firefox",
+        "request": "launch",
+        "url": "https://localhost:3000/",
+        "reAttach": true,
+        "pathMappings": [
+          {
+            "url": "webpack:///",
+            "path": "${workspaceFolder}/RetrospectiveExtension.Frontend/"
+          }
+        ]
+      }
+    ]
+  }
+  ```
+
+- Navigate to the '/RetrospectiveExtension.Frontend' folder, run `npm install` to download all the dependent packages listed in 'package.json'.
+
+- Run `npm run build:dev` to build the project.
+
+- Run `npm run start:dev`to start the webpack-dev-server
+
+- Start debugger (making sure the webpack-dev-server is still running). The default launch configuration should be set to Launch Firefox.
+
+- Once Firefox starts up, you should get an untrusted certificate error page. Select Advanced and then select **Accept the Risk and Continue** and log into your Azure DevOps account. From now on, if you leave this Firefox window open, the debugger will reattach instead of starting a clean Firefox instance each time.
+
+- Once you are logged in to Azure DevOps, your extension should be running. Set a breakpoint in a method in VS Code and you should see that breakpoint hit when that method executes.
```

Comment:
Should we hold off on merging this until we're able to diagnose the breakpoint issue?